src/cc/export/helpers.h

```diff
@@ -1243,10 +1260,10 @@ int bpf_usdt_readarg_p(int argc, struct pt_regs *ctx, void *buf, u64 len) asm("l
 #define lock_xadd(ptr, val) ((void)__sync_fetch_and_add(ptr, val))
 
 #define TRACEPOINT_PROBE(category, event) \
-int tracepoint__##category##__##event(struct tracepoint__##category##__##event *args)
+tracepoint__##category##__##event(struct tracepoint__##category##__##event *args)
 
 #define RAW_TRACEPOINT_PROBE(event) \
-int raw_tracepoint__##event(struct bpf_raw_tracepoint_args *ctx)
+raw_tracepoint__##event(struct bpf_raw_tracepoint_args *ctx)
```

Comment:
Is this change necessary ? If yes, I think docs should be updated accordingly.

I am afraid that tools using BCC might be broken due to this change.

Reply:
I agree with you, I do not want to do this change too, this commit is just a RFC patch,
because this change only makes it work well when expand {RAW_}TRACEPOINT_PROBE on LoongArch,
it needs to do some more analysis about the macro expansion.